lib/osf-components/addon/components/registries/schema-block-renderer/helper-text-icon/template.hbs

```diff
@@ -1,13 +1,18 @@
 {{#if this.helpText}}
-    <FaIcon
-        local-class='helper-text-icon'
-        tabindex='0'
-        @icon='question-circle'
-        @ariaHidden={{false}}
-        ...attributes
-    >
-        <BsPopover @triggerEvents={{array 'hover' 'focus'}}>
+    {{#let (unique-id 'helper-text-icon') as |id|}}
+        <FaIcon
+            local-class='helper-text-icon'
+            tabindex='0'
+            id={{id}}
+            @icon='question-circle'
+            @ariaHidden={{false}}
+            ...attributes
+        />
+        <BsPopover
+            @triggerEvents={{array 'hover' 'focus'}}
+            @triggerElement={{concat '#' id}}
+        >
             {{this.helpText}}
         </BsPopover>
-    </FaIcon>
+    {{/let}}
```

Comment:
Does this mean we can no longer use <FaIcon> as a block component?

Reply:
It doesn't look like it. Maybe there's a way to finagle it to do block component, but this seemed like the easiest way to work around it

Reply:
We have several other places in the codebase where we use FaIcon in block format. Those popovers might have stopped working as well.